app/models/enterprise.rb

```diff
@@ -462,6 +455,17 @@ def devise_mailer
 
   private
 
+  def validate_white_label_logo_link
+    return if white_label_logo.blank?
+
+    return if white_label_logo_link.blank?
+
+    uri = URI(white_label_logo_link)
+    self.white_label_logo_link = "http://#{white_label_logo_link}" if uri.scheme.nil?
```

Comment:
I would have expected `https` to be the default, but it doesn't really matter. If it matters, the user can update it!